recipes/recipes_emscripten/xeus-cpp/recipe.yaml

```diff
@@ -0,0 +1,49 @@
+context:
+  version: 0.3.0
+
+package:
+  name: xeus-cpp
+  version: '{{ version }}'
+
+source:
+  url: https://github.com/compiler-research/xeus-cpp/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 217d7870014327130902071c6e3027c62c0d97f356e0fca59a95376b1a6699e9
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - '{{ compiler("cxx") }}'
+    - cmake
+    - make  # [unix]
+  host:
+    - nlohmann_json
+    - xeus-lite
+    - xeus >=3.0.5,<4.0
+    - xtl >=0.7,<0.8
+    - llvm =16.0.6
```

Comment:
Use `CppInterOp` instead of `llvm`

Reply:
As this is for a build of 0.3.0 then changing to CppInterOp would be unnecessary. Will be needed 0.4.0 comes out. Also CppInterOp uses llvm 17, so this change will not work if xeus-cpp 0.3.0 is not compatible with llvm 17.